Cubical/Algebra/RingSolver/AlmostRing.agda

```diff
@@ -0,0 +1,73 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.Algebra.RingSolver.AlmostRing where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Algebra.Semigroup    hiding (⟨_⟩)
+open import Cubical.Algebra.Monoid       hiding (⟨_⟩)
+open import Cubical.Algebra.AbGroup   hiding (⟨_⟩)
+
+private
+  variable
+    ℓ : Level
+
+record IsAlmostRing {R : Type ℓ}
+                  (0r 1r : R) (_+_ _·_ : R → R → R) (-_ : R → R) : Type ℓ where
+
+  constructor isalmostring
+
+  field
+    +IsMonoid : IsMonoid 0r _+_
+    ·IsMonoid : IsMonoid 1r _·_
+    +Comm : (x y : R) → x + y ≡ y + x
+    ·Comm : (x y : R) → x · y ≡ y · x
+    RightDist : (x y z : R) → x · (y + z) ≡ (x · y) + (x · z)
```

Comment:
Should this be `·DistR+`?

Reply:
Yes, sure.

Reply:
I don't know though, if we agreed what left and right are. I think we didn't so I just use what I usually do.